.github/workflows/build-image.yaml

```diff
@@ -10,18 +10,37 @@ on:
 
 jobs:
     build-image:
-        # TODO: remove this, it's only needed for the frontend
-        strategy:
-            matrix:
-                group:
-                    - name: 'staging'
-                      if: true
-
-                    - name: 'prod'
-                      if: ${{ github.ref == 'refs/heads/master' }}
-
-        secrets: inherit
-        uses: ./.github/workflows/build-image-reusable.yaml
-        with:
-            if: ${{ matrix.group.if }}
-            name: ${{ matrix.group.name }}
+        runs-on: ubuntu-latest
+        env:
+            CAN_PUSH: "${{ secrets.DOCKER_PASSWORD != ' && secrets.DOCKER_USERNAME != ' }}"
```

Comment:
how does thi work? missing a quote no? `secrets.DOCKER_PASSWORD != ''`

Reply:
good question, it was already like this and working so I guess github actions is flexible. Will fix :p